CHANGELOG.md

```diff
@@ -1,79 +1,82 @@
+* [ENHANCEMENT] Add histograms `spans_distance_in_future_seconds` / `spans_distance_in_past_seconds` that count spans with end timestamp in the future / past. While spans in the future are accepted, they are invalid and may not be found using the Search API. [#4936](https://github.com/grafana/tempo/pull/4936) (@carles-grafana)
+* [ENHANCEMENT] Add support for scope in cost-attribution usage tracker. [#5646](https://github.com/grafana/tempo/pull/5646) (@electron0zero)
+* [ENHANCEMENT] Add alert for high error rate reported by vulture. [#5206](https://github.com/grafana/tempo/pull/5206) (@ruslan-mikhailov)
+* [ENHANCEMENT] Support the new `db.namespace` attribute for service-graph DB nodes. [#5602](https://github.com/grafana/tempo/pull/5602) (@gouthamve)
+* [ENHANCEMENT] TraceQL metrics performance increase for simple queries. [#5247](https://github.com/grafana/tempo/pull/5247) (@mdisibio)
+* [ENHANCEMENT] TraceQL search and metrics performance increase. [#5280](https://github.com/grafana/tempo/pull/5280) (@mdisibio)
+* [ENHANCEMENT] TraceQL performance improvement. [#5218](https://github.com/grafana/tempo/pull/5218) (@mdisibio)
+* [ENHANCEMENT] TraceQL `compare()` performance improvement. [#5419](https://github.com/grafana/tempo/pull/5419) (@mdisibio)
+* [ENHANCEMENT] Align traceql attribute struct for better performance. [#5240](https://github.com/grafana/tempo/pull/5240) (@mdisibio)
+* [ENHANCEMENT] Drop invalid prometheus label names in the `spanmetrics` processor. [#5122](https://github.com/grafana/tempo/pull/5122) (@KyriosGN0)
+* [ENHANCEMENT] Improve logging and tracing in the write path to include tenant info. [#5436](https://github.com/grafana/tempo/pull/5436) (@javiermolinar)
+* [ENHANCEMENT] Added usage tracker example. [#5356](https://github.com/grafana/tempo/pull/5356) (@javiermolinar)
+* [ENHANCEMENT] Add Stop method. [#5293](https://github.com/grafana/tempo/pull/5293) (@stephanos)
+* [ENHANCEMENT] Use peer attributes to determine the name of a client service virtual node in the service graph. [#5381](https://github.com/grafana/tempo/pull/5381) (@martenm)
+* [ENHANCEMENT] Put actual size for writing to backend. [#5413](https://github.com/grafana/tempo/pull/5413) (@ruslan-mikhailov)
+* [ENHANCEMENT] Upgrade Azurite and Fake-gcs-server to latest version. [#5512](https://github.com/grafana/tempo/pull/5512) (@javiermolinar)
+* [ENHANCEMENT] Make block ordering deterministic. [#5411](https://github.com/grafana/tempo/pull/5411) (@rajiv-singh)
+* [ENHANCEMENT] Improve exemplar selection in `quantile_over_time()`. [#5278](https://github.com/grafana/tempo/pull/5278) (@zalegrala)
+* [ENHANCEMENT] Add live store to `jsonnet` lib. [#5591](https://github.com/grafana/tempo/pull/5591) [#5606](https://github.com/grafana/tempo/pull/5606) [#5609](https://github.com/grafana/tempo/pull/5609) (@mapno)
+* [ENHANCEMENT] Measure bytes received before limits. [#5601](https://github.com/grafana/tempo/pull/5601) (@mapno)
```

Comment:
nit: maybe we can mention the metric names as well, like we do in other chanelog entries where we added or removed a metric.